sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.netstandard2.0.cs

```diff
@@ -1,5 +1,14 @@
 namespace Azure
 {
+    public partial class MessageWithMetadata
+    {
+        public MessageWithMetadata() { }
+        public virtual Azure.Core.ContentType? ContentType { get { throw null; } set { } }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        protected virtual Azure.Core.ContentType? ContentTypeCore { get { throw null; } set { } }
```

Comment:
Should we make this a method instead?

Reply:
I don't have a strong opinion either way.   In my opinion, a good comment on whichever members we use that mention the reason why this exists would be very helpful for contributors, both internal and external.    We can see how it's used locally by just looking at references in this class, but to really understand that it exists to allow a type change in the base is less obvious without spelunking.